spec/integration/user_can_visit_business_index_page_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.feature "user can view business index" do
+  scenario "user visits home page and sees the business index link" do
+    visit root_url
+
+    expect(page).to have_link("Business Index")
+  end
+
+  scenario "user clicks 'Business Index' link on home page" do
+    businesses = create_list(:business, 3)
+    visit root_url
+
+    click_link "Business Index"
+    expect(current_path).to eq(businesses_path)
```

Comment:
let's hardcode in the path: expect(path).to eq('/"#business.name"') 
so we can make sure the vanity route is displaying the way we want